interface/forms/fee_sheet/new.php

```diff
@@ -1,5 +1,5 @@
 <?php
-/**
+    /**
  * Fee Sheet Program used to create charges, copays and add diagnosis codes to the encounter
  *
```

Comment:
This script is missing following prior code changes:
https://github.com/zbig01/openemr/commit/0818d61f5df2669892e948b4c47537d4d9f4043e#diff-80ab3fa9f3a8ce5754da82b010ee97ec (just line 1191 in your current script)

(after above issue and other issues btought up in thei script are addressed, then this script will be ready to go.

Reply:
Looks like this was missed in recent commit

Reply:
confirmed fixed